packages/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-settings-plugin/src/main/kotlin/expo/modules/plugin/ExpoAutolinkingSettingsExtension.kt

```diff
@@ -27,6 +35,30 @@ open class ExpoAutolinkingSettingsExtension(val settings: Settings) {
    */
   var exclude: List<String>? = null
 
+  /**
+   * The file pointing to the React Native Gradle plugin.
+   */
+  val reactNativeGradlePlugin: File by lazy {
+    File(
+      settings.providers.exec { env ->
+        env.workingDir(settings.rootDir)
+        env.commandLine("node", "--print", "require.resolve('@react-native/gradle-plugin/package.json')")
```

Comment:
```suggestion
        env.commandLine("node", "--print", "require.resolve('@react-native/gradle-plugin/package.json', { paths: [require.resolve('react-native/package.json')] })")
```

same as https://github.com/expo/expo/pull/35790#discussion_r2020889673